src/main/java/org/isf/admission/gui/AdmittedPatientBrowser.java

```diff
@@ -210,7 +210,9 @@ public void deleteAdmissionUpdated(AWTEvent e) {
 					//the patient is admitted
 					if (elemAdm.getId() == adm.getId())
 						//same admission --> delete
-						elem.setAdmission(null);	
+					{
```

Comment:
maybe bracket on same line of if?

Reply:
I got carried away and fixed many more bracket/spacing issues.   Also found a few more names to change.